doc/supported_operations.md

```diff
@@ -0,0 +1,16 @@
+# Containerized Data Importer supported operations
+The Containerized Data Importer (CDI) supports importing data/disk images from a variety of sources, and it can be hard to understand which combination of operations is supported and if that operation requires scratch space. This document explains the combinations that are supported and which ones require scratch space. All Kubevirt content type operations will support both file system Data Volumes (DV) as well as block volume DVs.
+
+## Supported matrix
+
+| Content-Type | http | https | http basic auth | Registry | Upload | S3 Bucket  |
+|--------------|---------|-|--|-------|--------|------------|
+| KubeVirt(QCOW2)        |<ul><li>[x] QCOW2</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |<ul><li>[x] QCOW2\*\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |<ul><li>[x] QCOW2</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> | <ul><li>[ ] QCOW2</li><li>[ ] GZ</li><li>[ ] XZ</li><li>[ ] TAR</li></ul> | <ul><li>[x] QCOW2\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> | <ul><li>[x] QCOW2\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |
+| KubeVirt (RAW)          |<ul><li>[x] RAW</li><li>[x] GZ</li><li>[x] XZ</li><li>[x] TAR</li></ul> |<ul><li>[x] RAW</li><li>[x] GZ</li><li>[x] XZ</li><li>[x] TAR</li></ul> |<ul><li>[x] RAW</li><li>[x] GZ</li><li>[x] XZ</li><li>[x] TAR</li></ul> | <ul><li>[x] RAW*</li><li>[ ] GZ</li><li>[ ] XZ</li><li>[ ] TAR</li></ul> | <ul><li>[x] RAW</li><li>[x] GZ</li><li>[x] XZ</li><li>[x] TAR</li></ul> | <ul><li>[x] RAW</li><li>[x] GZ</li><li>[x] XZ</li><li>[x] TAR</li></ul> |
+| Archive+ | <ul><li>[x] TAR</li></ul> | <ul><li>[x] TAR</li></ul> | <ul><li>[x] TAR</li></ul> | <ul><li>[ ] TAR</li></ul> | <ul><li>[ ] TAR</li></ul> | <ul><li>[ ] TAR</li></ul> |
+
+\* Requires scratch space
```

Comment:
Could we add a paragraph reminding people what this means in practice?  Ie.  A scratch PVC of the same size as the target PVC will be allocated and removed automatically and is used to store temporary data as the imported file is converted for use with kubevirt.

Reply:
I added links to the scratch space document, as well as clearing up some stuff in that document.